templates/navbar.html

```diff
@@ -24,9 +24,9 @@
 					</ul>
 					<ul class="nav navbar-nav navbar-right">
 						<li class="{% if page == 'new' %}active{% endif %}">
-							<a href="/new">
-								[+] New Paste
-							</a>
+							<button type="button" class="btn btn-primary" style="margin-top: 5px;" onclick="location.href='/new';">
+      							<span class="glyphicon glyphicon-plus"></span> new paste
```

Comment:
Unfortunately the `glyphicon` font does not work since the Tor Browser does **not** load custom fonts. This is why I added the `[+]` button in the first place. In addition to that, we want to avoid inline stylesheets (`style=""`) as well as JavaScript (`onclick=""`). Currently PR #25 addresses this issue by using native bootstrap. 

Thanks for the Pull Request!